pkgs/applications/virtualization/xen/4.16/0001-xen-fig-geneneration-4.16.patch

```diff
@@ -0,0 +1,16 @@
+Remove an unecessary pipe that was causing
```

Comment:
Nit: “unnecessary” (but I won’t block the merge on it).

Reply:
Fixed.